src/app/component/app-nav-bar/app-nav-bar.component.ts

```diff
@@ -73,7 +73,7 @@ export class AppNavBarComponent implements OnInit, OnChanges {
 
     })
 
-    if ((window.innerWidth < 600) && window.location.href.includes('/public/home')) {
+    if ((window.innerWidth < 600) && (this.configSvc.userProfile === null)) {
```

Comment:
can we can not check by isXSmall$ instead of checking innerWidth?

Reply:
Yes, Changed